Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py

```diff
@@ -627,6 +630,44 @@ def import_ioc_with_approval(import_type, import_value, confidence="50", classif
         return_outputs("The data was not imported. Check if valid arguments were passed", None)
 
 
+def import_ioc_without_approval(file_id, classification, confidence=None, allow_unresolved=None,
+                                source_confidence_weight=None, expiration_ts=None, severity=None,
+                                tags=None, trustedcircles=None):
+    """
+        Imports indicators data to ThreatStream.
+        file_id of uploaded file to war room or URL. Other fields are
+    """
+    if allow_unresolved:
+        allow_unresolved = allow_unresolved == 'yes'
+    if tags:
+        tags = argToList(tags)
+    if trustedcircles:
+        trustedcircles = argToList(trustedcircles)
+    try:
+        # entry id of uploaded file to war room
+        file_info = demisto.getFilePath(file_id)
+        with open(file_info['path'], 'rb') as uploaded_file:
+            ioc_to_import = json.load(uploaded_file)
+    except json.JSONDecodeError:
+        raise DemistoException(F"Entry {file_id} does not contain a valid json file.")
+    except Exception:
+        raise DemistoException(F"Entry {file_id} does not contain a file.")
+    ioc_to_import.update({'meta': assign_params(
+        classification=classification,
+        confidence=confidence,
+        allow_unresolved=allow_unresolved,
+        source_confidence_weight=source_confidence_weight,
+        expiration_ts=expiration_ts,
+        severity=severity,
+        tags=tags,
+        trustedcircles=trustedcircles
+    )})
+
+    params = build_params()
+    res = http_request("PATCH", "v1/intelligence/", params=params, json=ioc_to_import, text_response=True)
+    return_outputs("The data was imported successfully.", {}, res)
```

Comment:
just curious, what is the response here?

Reply:
In a normal response, it's expected to be empty.
![image](https://user-images.githubusercontent.com/20818773/93169813-9cd54400-f72e-11ea-8651-1996e80de557.png)